pure.zsh

```diff
@@ -27,24 +27,26 @@
 # 165392 => 1d 21h 56m 32s
 # https://github.com/sindresorhus/pretty-time-zsh
 prompt_pure_human_time() {
-	echo -n " "
-	local tmp=$1
-	local days=$(( tmp / 60 / 60 / 24 ))
-	local hours=$(( tmp / 60 / 60 % 24 ))
-	local minutes=$(( tmp / 60 % 60 ))
-	local seconds=$(( tmp % 60 ))
-	(( $days > 0 )) && echo -n "${days}d "
-	(( $hours > 0 )) && echo -n "${hours}h "
-	(( $minutes > 0 )) && echo -n "${minutes}m "
-	echo "${seconds}s"
+	local human=" " var=$1 unix=$2
+	local days=$(( unix / 60 / 60 / 24 ))
+	local hours=$(( unix / 60 / 60 % 24 ))
+	local minutes=$(( unix / 60 % 60 ))
+	local seconds=$(( unix % 60 ))
+	(( days > 0 )) && human+="${days}d "
+	(( hours > 0 )) && human+="${hours}h "
+	(( minutes > 0 )) && human+="${minutes}m "
+	human+="${seconds}s"
+	eval "${var}=\$human"
 }
```

Comment:
Would love a PR on https://github.com/sindresorhus/pretty-time-zsh improving it too :heart: :)